lib/matplotlib/figure.py

```diff
@@ -1129,6 +1138,10 @@ def add_axes(self, *args, **kwargs):
         polar : bool, default: False
             If True, equivalent to projection='polar'.
 
+        axes_class : subclass type of `~.axes.Axes`, optional
+            The `.axes.Axes` subclass that is instantiated.  This parameter
+            is incompatible with *projection* and *polar*.
```

Comment:
This is somewhat mysterious.  Can we link to an example here?

Reply:
do you want to pick one? :)

Reply:
I dunno, which ever subclass gets used the most?

Reply:
I found the right thing to link.